packages/flutter/lib/src/material/dialog.dart

```diff
@@ -176,6 +177,43 @@ class AlertDialog extends StatelessWidget {
   }
 }
 
+/// A button used in a [SimpleDialog].
```

Comment:
I added more description and renamed the class to `SimpleDialogOption`, which is more accurate.